docs/docs/components/style-spec/item.js

```diff
@@ -0,0 +1,368 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import md from '../md';
+import { highlightJSON } from '../prism_highlight';
+import entries from 'object.entries';
+import SDKSupportTable from '../sdk_support_table';
+import Icon from '@mapbox/mr-ui/icon';
+import Property from './property.js';
+import Subtitle from './subtitle.js';
+
+export default class Item extends React.Component {
+    type(spec = this.props, plural = false) {
+        switch (spec.type) {
+            case null:
+            case '*':
+                return;
+            case 'light':
+                return (
+                    <span>
+                        {' '}
+                        <a href="/maplibre-gl-style-spec/style-spec/light/">
+                            light
+                        </a>
+                    </span>
+                );
+            case 'transition':
+                return (
+                    <span>
+                        {' '}
+                        <a href="/maplibre-gl-style-spec/style-spec/transition/">
+                            transition
+                        </a>
+                    </span>
+                );
+            case 'sources':
```

Comment:
Also just a poor decision to replicate the style spec here in the UI... This means that `terrain` and `padding` is probably not here...